### PR TITLE
add react and jest as dependencies and add node-modules to .gitignore

### PR DESCRIPTION
**Pull request by peterianmuller**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,9 +17,12 @@
   "bugs": {
     "url": "https://github.com/peterianmuller/quiz-library/issues"
   },
-  "homepage": "https://github.com/peterianmuller/quiz-library#readme",
-  "dependencies": {},
+  "homepage": "https://github.com/peterianmuller/quiz-library#readme",
```
Comment (svey) [inline package.json:20]: blocking: this is no longer true; pim => 16b; same for url attr above. let's update it throughout.
Could be a separate pr if you want.

**Comment by peterianmuller** [comment]
Also for changing the package.json on master, won't merging this one into master change the url attr in the package.json in the master to be this updated one?
